NEWS.md

```diff
@@ -4,6 +4,11 @@
 
  - Package Extension guidance on the front page was updated and moved to [this location](https://pharmaverse.github.io/admiraldev/main/articles/package_extensions.html) in `{admiraldev}`, and a link was added to point to the new location (#61).
```

Comment:
This link doesn't work for me, but [this one does].

Reply:
I don't know much about what you want to refer from admiraldev here exactly, maybe we can just copy paste what we have from admiral readme ?

Reply:
(@rossfarrugia / @manciniedoardo : also just fixed the templates - we were using old `admiral.test` => I updated news.md as well on this

Reply:
good spot! i think the admiraldev reference is fine for now

Reply:
oh sorry missed the broken link comment - @dgrassellyb you just need to remove the `main/` bit in that link and it works then

Reply:
@bms63 @manciniedoardo didn't we get rid of the `README.Rmd` approach now and we only use `README.md`? sorry, every boulder we turn over seems to find a new surprise here, as we've updated so much in admiral conventions since the last time we made a new pkg ext.

Reply:
admiral never had a `.Rmd` it was adopted into the other extension packages and admiraldev right about when I came onbaord.  Honestly, I'd like to get rid of them all - I think we had them for helping with dynamic linking but can't remember anymore??

Anyways, I think we should remove here and remove in `peds` as well and maybe make a push for removing them in all packages??

 Try to keep as simple as possible!

Reply:
just deleted it, I think as well it's a bit overkill honestly - for `vbump` I commented it to not use it for `admiraltemplates` directly, but still keep it because some other projects might want to use it

Reply:
i just removed from admiralpeds too